source/History.js

```diff
@@ -353,6 +366,9 @@
 
 			switch (state) {
 			case 'empty':
+				if (history.state.currentObjId == 'historyMaster') {
```

Comment:
Hi @DavidUM, it seems that `history.state` can be `null` here, if we do not have any previous history i.e. this is the first page we load in the browser. I think we should add logic to handle this case properly, which may involve adding guard code. Thanks!